app/service/file_svc.py

```diff
@@ -97,15 +97,21 @@ def __init__(self, plugins, exfil_dir):
         self.special_payloads[name] = func
 
     @staticmethod
-    async def compile_go(platform, output, src_fle):
+    async def compile_go(platform, output, src_fle, ldflags=None):
```

Comment:
alternatively we could do 
```
    async def compile_go(platform, output, src_fle, ldflags='-s -w'):
```
probably depends on how hard we want to enforce using the `-s -w` flags